elements/pl-matrix-output/pl-matrix-output.py

```diff
@@ -6,19 +6,37 @@
 
 def prepare(element_html, data):
     element = lxml.html.fragment_fromstring(element_html)
-    pl.check_attribs(element, required_attribs=[], optional_attribs=['digits'])
+    required_attribs = []
+    optional_attribs = ['digits', 'display-matlab-tab', 'display-mathematica-tab', 'display-python-tab']
```

Comment:
Changed this, left the internal html_params names as they were, not sure if they should also be "show_matlab" instead of "display_matlab_tab" for consistency

Reply:
I’m inclined to also change the internal `html_params` names. The less changing of names we have the better (although the internal names should be snake_case).